City-of-Helsinki/varaamo
# VAR-429, VAR-432 | Implement sorting resources by accessibility

## Description :sparkles:

Adds sorting by accessibility to the resource search page.

This commit additionally connects the selected accessibility preferences
to the redux controlled state. This enabled to e.g. update the
resource listing's accessibility icons instantly as preferences are
changed.

### Closes :no_good_woman:

[VAR-429](https://helsinkisolutionoffice.atlassian.net/browse/VAR-429)
[VAR-432](https://helsinkisolutionoffice.atlassian.net/browse/VAR-432)

## Testing :alembic:

### Automated tests :gear:️

`app/state/reducers/__tests__/accessibilityReducer.test.js `

### Manual testing :construction_worker_man:

- On the front page, search for resources.
- Once the results are listed, toggle sorting by accessibility from the dropdown menu.
- The resources should be refetched, with resources that have shortcomings for your accessibility preferences listed last.
